### DISCUSSION TITLE
PVP outpost spawn in as ruin in ruin mission

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#15098] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

PVP outpost will spawn as ruin in a ruin mission and not an actual ruin

### Reproduction steps

[ruinnotruin.zip](https://github.com/user-attachments/files/17690681/ruinnotruin.zip)

Load the save file and travel to marked ruin

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
No issues found in SP or MP ruins/outpost PVP
Closing discussion as resolved.
